src/bid/encoding.rs

```diff
@@ -75,10 +76,8 @@ impl Into<StorageScalar> for StorageBid {
         words_deposit.push(BlsScalar::from(self.expiration_ts as u64));
 
         // Push both JubJubAffine coordinates as a Scalar.
-        words_deposit.push(self.encrypted_value.0.get_x());
-        words_deposit.push(self.encrypted_value.0.get_y());
-        words_deposit.push(self.encrypted_blinder.1.get_x());
-        words_deposit.push(self.encrypted_blinder.1.get_y());
+        words_deposit.extend_from_slice(self.encrypted_value.cipher());
+        words_deposit.extend_from_slice(self.encrypted_blinder.cipher());
```

Comment:
We should change `type_fields` since now encrypted_value and `encrypted_blinder` are `BlsScalar` instead of `AffinePoint`

Reply:
Fixed